modules/opensimplex/noise.cpp

```diff
@@ -0,0 +1,226 @@
+#include "noise.h"
```

Comment:
Please add Godot's copyright header.

I'd also suggest to rename `noise.{cpp,h}` to `simplex_noise.{cpp,h}` to match the class name.